internal/tiltfile/config/resources.go

```diff
@@ -44,8 +45,20 @@ func setEnabledResources(thread *starlark.Thread, fn *starlark.Builtin, args sta
 	return starlark.None, nil
 }
 
+func clearEnabledResources(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	err := starkit.SetState(thread, func(settings Settings) Settings {
```

Comment:
we probably should still do an Unpack() here, so that it errors out if the user passes any arguments.

Reply:
oops, yes!